uniq/src/cli.rs

```diff
@@ -0,0 +1,53 @@
+use clap::{
+    crate_authors, crate_description, crate_name, crate_version, App, AppSettings::ColoredHelp, Arg,
+};
+
+pub(crate) fn create_app<'a, 'b>() -> App<'a, 'b> {
+    App::new(crate_name!())
+        .version(crate_version!())
+        .author(crate_authors!())
+        .about(crate_description!())
+        .help_message("Display help information.")
+        .version_message("Display version information.")
+        .help_short("?")
+        .settings(&[ColoredHelp])
+        .arg(
+            Arg::with_name("INPUT")
+                .help("Input file (defaults to stdin).")
+                .index(1)
+                .default_value("-"),
+        )
+        .arg(
+            Arg::with_name("OUTPUT")
+                .help("Output file (defaults to stdout).")
```

Comment:
I think we could have a help better message here, like: "Output file to write, or '-' to write from standard output."